src/data/blog/posts/2022-02-10-unison-2021-year-in-review/index.md

```diff
@@ -0,0 +1,110 @@
+---
+slug: unison-2021-year-in-review
+title: "Unison in 2021, 2022 and beyond: year in review and future plans"
+description: 2021 was a big year for Unison. 
+date: 2022-02-10
+authors: ["runar-bjarnason"]
+categories: ["news"]
+featuredImage: https://unison-lang.org/img/posts/unison-2021.svg
+---
+
+2021 was a big year for Unison. We released a new alpha version roughly every other month, and each release came with some serious improvements. In total, we merged more than 600 pull requests. Here are some of the highlights.
+
+## A beautiful GUI for browsing the Unison codebase
+
+(embed gif of codebase UI here)
```

Comment:
Do we want to do this before merging?

Reply:
Yes! sorry I should have added to the description of the PR or made it draft. This was mostly to see how the hero image would work on the old blog

Reply:
Added the video 🎉